cpp/include/fixed_point/fixed_point.hpp

```diff
@@ -20,8 +20,7 @@
 #include <iostream>
 #include <limits>
 
-namespace cudf {
-namespace fixed_point {
+namespace fixpt {
```

Comment:
Why the unnecessary abbreviation?

Reply:
Talked about this with @jrhemstad on slack. We moved `fixed_point` out of `cudf` namespace and then the type alias for the namespace in the test file became `using namespace fixed_point` instead of `cudf::fixed_point` and it resulted in a compiler ambiguity error:
```cpp
In file included from /home/choekstra/ws3/cudf/cpp/tests/fixed_point/fixed_point_tests.cpp:19:0:
/home/choekstra/ws3/cudf/cpp/include/fixed_point/fixed_point.hpp:23:11: note: candidates are: namespace fixed_point { }
 namespace fixed_point {
           ^~~~~~~~~~~
/home/choekstra/ws3/cudf/cpp/include/fixed_point/fixed_point.hpp:103:7: note:                 template<class Rep, fixed_point::Radix Rad> class fixed_point::fixed_point
 class fixed_point {
```
So then I decided to see if there was a abbreviation generator out there and lo and behold:
https://www.abbreviations.com/acronym-generator/fixed%20point
I personally liked `fpt` the best as a non-`fp` option, but that could still stand for floating point. `fixp` was the next best so I went with `fixpt`. Jake thought it was ok.

Reply:
Isn't moving it outside the cudf namespace putting the cart before the horse?

Reply:
Sorry to nag on this, but I take naming very seriously.

Reply:
To quote Kate Gregory, "[naming is hard](https://youtu.be/MBRoCdtZOYg)" - and we should all take naming very seriously. 

I didn't realise this was the game plan. From the stand up last week (Jan 29) is sounded like their were potential customers already waiting on this which gave me the impression that we wanted it out of the namespace before we merge. But if you the plan is what you mentioned I am totally OK with keeping it in `cudf` namespace for now and moving it out later.

@jrhemstad thoughts?

Reply:
Wouldn't `using namespace cudf::fixed_point` have the same problem? I think the issue ultimately originates using the same name for a namespace and a type.

Reply:
Somehow the compiler is able to figure out that `using namespace cudf::fixed_point` refers to the namespace. I am sure that if you did:
```cpp
using namespace cudf;
using namespace fixed_point;
```
then you would have the same issue.
...
I just checked and that is the case:
https://godbolt.org/z/QVa39C
```cpp
namespace A {
namespace B {
    class B {};
}}

using namespace A::B; // OK

// using namespace A;
// using namespace B; // not OK

int main() {
    using alias = B;
    return 0;
}
```

Reply:
Are there planned to be other (externally referenced) names in the `fixed_point` namespace other than `fixed_point`?

Reply:
To my knowledge the plan (or in the current pipeline) is just `fixed_point`, however it could be extended if there are other facilities that we might want to put under the namespace/umbrella. @jrhemstad would be able to better answer this though.

Reply:
We could preemptively call the namespace something broader, like `numeric`, or `num`.

Reply:
We talked in standup and decided `numeric` outside of `cudf` is the way to go.